packages/pds/src/api/com/atproto/handle.ts

```diff
@@ -60,9 +78,34 @@ export default function (server: Server, ctx: AppContext) {
           }
           throw err
         }
-
         await ctx.plcClient.updateHandle(requester, handle, ctx.keypair)
       })
     },
   })
 }
+
+const resolveExternalHandle = async (
```

Comment:
I'd rather race these, but I was having a hard time doing it that didn't leave the second promise dangling & we don't have access to piping an abortcontroller through the ApiAgent

Reply:
Curious about this— what do you mean by "dangling" here?  Still, DNS should be quick, so I agree this is probably okay :)

Reply:
P.S. really would be nice to support aborts in the client lib!

Reply:
basically: we don't wait for the promise to resolve, so the test runner gets mad at me because we have some async code still running after test ends

only way I can think to get out of that is with aborts

agreed on supporting them 👍